test/integration/queue/consumers/ManagedConsumer-test.js

```diff
@@ -22,6 +22,8 @@ const testMessage = new Event({
   }
 })
 
+const testMessageCleaned = testMessage.clean()
```

Comment:
Should this be outside of the beforeEach? It seems like both testMessage and testMessageCleaned should be inside the beforeEach to help avoid unexpected modifications between tests in the future. It looks like a similar pattern could be used in other tests.

Reply:
I agree, we should pull the creation of both `testMessage` and `testMessageCleaned` into the `beforeEach`.  To ensure that tests don't make bad assertions, we should probably randomly generate the content (`compare`) of the message body.